docs/articles/expensify-classic/connections/sage-intacct/Connect-To-Sage-Intacct.md

```diff
@@ -126,8 +132,14 @@ To enable Customization Services go to **Company > Subscriptions > Customization
 ### Step 6: Create a Test Workspace in Expensify and Download the [Expensify Package](https://www.expensify.com/tools/integrations/downloadPackage)
 Creating a test workspace in Expensify allows you to have a sandbox environment for testing before implementing the integration live. If you are already using Expensify, creating a test workspace ensures that your existing group workspace rules and approval workflows remain intact. Here's how to set it up:
 1. Go to **expensify.com > Settings > Workspaces > New Workspace**.
+
+![Image of creating a new Workspace in Expensify]({{site.url}}/assets/images/SageConnectCreatingWorkspace.png){:width="100%"}
+
 2. Name the workspace something like "Sage Intacct Test Workspace."
 3. Go to **Connections > Sage Intacct > Connect to Sage Intacct**.
+
+![Image of selecting the Sage Intacct integration in Expensify]({{site.url}}/assets/images/SageConnectEnableSage.png){:width="100%"}
+
 4. Select **Download Package** (You only need to download the file; we'll upload it from your Downloads folder later).
```

Comment:
When this renders the numbering is off, it restarts after each image. You could try fixing this by indenting the images 4 spaces, or by using html tags for an [ordered list](https://www.w3schools.com/html/html_lists_ordered.asp). Try the indentation first. Lmk if you need help.


![Google Chrome 2024-11-19 15 13 47](https://github.com/user-attachments/assets/14183394-17b2-4203-b7ea-390b78271c03)

Reply:
The numbering is still off. I'll play around with it locally and see if I can get it fixed.